### ISSUE TITLE
Try pre-trained NN

### ISSUE THREAD

**Issue by dzenanz** [OpenImaging/miqa#24]
For example, from MONAI male/female classification example.

**Comment by dzenanz** [comment]
Without pretraining:
train_confusion_matrix:
[[81  0]
 [ 2 61]]
val_confusion_matrix:
[[ 2  5]
 [ 6 71]]
              precision    recall  f1-score   support
           0       0.25      0.29      0.27         7
           1       0.93      0.92      0.93        77
    accuracy                           0.87        84
   macro avg       0.59      0.60      0.60        84
weighted avg       0.88      0.87      0.87        84

With pretraining:

epoch 22 average loss: 0.0438
confusion matrix:
[[45 19]
 [ 5 75]]
Evaluating on validation set
....................................................................................
val_confusion_matrix:
[[ 3  4]
 [24 53]]
              precision    recall  f1-score   support

           0       0.11      0.43      0.18         7
           1       0.93      0.69      0.79        77

    accuracy                           0.67        84
   macro avg       0.52      0.56      0.48        84
weighted avg       0.86      0.67      0.74        84

saved new best metric model
current epoch: 22 current AUC: 0.5584 best AUC: 0.5584 at epoch 22
Learning rate after epoch 22: 8.36716218448071e-06

**Comment by dzenanz** [comment]
As expected, this does not help.

**Comment by dzenanz** [comment]
Pretraining weights: [pretrained.zip](https://github.com/OpenImaging/miqa/files/7007891/pretrained.zip)
